tests/src/drivers/test_driver_pake.c

```diff
@@ -0,0 +1,319 @@
+                operation, step, input, input_length );
+#elif defined(MBEDTLS_PSA_BUILTIN_PAKE)
+        mbedtls_test_driver_pake_hooks.driver_status =
+            mbedtls_psa_pake_input(
+                operation, step, input, input_length );
+#else
+        (void) operation;
+        (void) step;
+        (void) input;
+        (void) input_length;
+        mbedtls_test_driver_pake_hooks.driver_status = PSA_ERROR_NOT_SUPPORTED;
+#endif
+    }
+
+    return( mbedtls_test_driver_pake_hooks.driver_status );
+}
+
+psa_status_t mbedtls_test_transparent_pake_get_implicit_key(
+                mbedtls_transparent_test_driver_pake_operation_t *operation,
+                uint8_t *output, size_t *output_size )
+{
+    mbedtls_test_driver_pake_hooks.hits++;
+
+    if( mbedtls_test_driver_pake_hooks.forced_status != PSA_SUCCESS )
+    {
+         mbedtls_test_driver_pake_hooks.driver_status =
+             mbedtls_test_driver_pake_hooks.forced_status;
+    }
+    else
+    {
+#if defined(MBEDTLS_TEST_LIBTESTDRIVER1) && \
+    defined(LIBTESTDRIVER1_MBEDTLS_PSA_BUILTIN_PAKE)
+        mbedtls_test_driver_pake_hooks.driver_status =
+            libtestdriver1_mbedtls_psa_pake_get_implicit_key(
+                operation,  output, output_size );
+#elif defined(MBEDTLS_PSA_BUILTIN_PAKE)
+        mbedtls_test_driver_pake_hooks.driver_status =
+            mbedtls_psa_pake_get_implicit_key(
+                operation, output, output_size );
+#else
+        (void) operation;
+        (void) output;
+        mbedtls_test_driver_pake_hooks.driver_status = PSA_ERROR_NOT_SUPPORTED;
+#endif
+    }
+
+    return( mbedtls_test_driver_pake_hooks.driver_status );
+}
+
+psa_status_t mbedtls_test_transparent_pake_abort(
+                mbedtls_transparent_test_driver_pake_operation_t * operation )
+{
+    mbedtls_test_driver_pake_hooks.hits++;
+
+    if( mbedtls_test_driver_pake_hooks.forced_status != PSA_SUCCESS )
+    {
+         mbedtls_test_driver_pake_hooks.driver_status =
+             mbedtls_test_driver_pake_hooks.forced_status;
+    }
+    else
+    {
+#if defined(MBEDTLS_TEST_LIBTESTDRIVER1) && \
+    defined(LIBTESTDRIVER1_MBEDTLS_PSA_BUILTIN_PAKE)
+        mbedtls_test_driver_pake_hooks.driver_status =
+            libtestdriver1_mbedtls_psa_pake_abort(
+                operation );
+#elif defined(MBEDTLS_PSA_BUILTIN_PAKE)
+        mbedtls_test_driver_pake_hooks.driver_status =
+            mbedtls_psa_pake_abort(
+                operation );
+#else
+        (void) operation;
+        mbedtls_test_driver_pake_hooks.driver_status = PSA_ERROR_NOT_SUPPORTED;
+#endif
+    }
+
+    return( mbedtls_test_driver_pake_hooks.driver_status );
+}
+
+/*
+ * opaque versions, to do
+ */
+psa_status_t mbedtls_test_opaque_pake_setup(
+                mbedtls_opaque_test_driver_pake_operation_t *operation,
+                const psa_crypto_driver_pake_inputs_t *inputs )
+{
+    (void) operation;
+    (void) inputs;
+    return( PSA_ERROR_NOT_SUPPORTED );
```

Comment:
Are these `opaque` functions going to be implemented in this PR? Or will they be part of another one?

Reply:
This is out of scope for this PR.